lib/redux/reducers/users.js

```diff
@@ -222,11 +222,14 @@ export function uploadTargetUser(state = null, action) {
     case actionTypes.LOGIN_SUCCESS:
     case actionTypes.SET_USER_INFO_FROM_TOKEN:
       const { user, profile, memberships } = action.payload;
+      const uploadMemberships = _.filter(memberships, (mship) => {
+        return !_.isEmpty(_.get(mship, ['profile', 'patient']));
+      });
```

Comment:
Cool. Didn't know you could use an array of terms **or** a string to represent path to variable you want to `_.get()` e.g. 'profile.patient'.